lib/down/net_http.rb

```diff
@@ -216,7 +216,9 @@ def net_http_request(uri, options, follows_remaining:, &block)
         request_error!(exception)
       end
 
-      if response.is_a?(Net::HTTPRedirection)
+    if response.is_a?(Net::HTTPNotModified)
+      raise Down::NotModified
+    elsif response.is_a?(Net::HTTPRedirection)
```

Comment:
I think the indentation here got munged
```suggestion
      if response.is_a?(Net::HTTPNotModified)
        raise Down::NotModified
      elsif response.is_a?(Net::HTTPRedirection)
```